### PR TITLE
Create console app

### PR DESCRIPTION
**Pull request by killy10o10**
## In This feature I completed the following requirements: 

- **Create a main.rb file that will serve as your console app entry point. It should have methods that do the following:**

1. >List all books.
2. > List all people.
3. > Create a person (teacher or student, not a plain Person).
4. > Create a book.
5. > Create a rental.
6. > List all rentals for a given person id.

- **In your main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method should do the following:**

1. > Present the user with a list of options to perform.
2. > Lets users choose an option.
3. > If needed, ask for parameters for the option.
4. > Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+
+    student = Student.new(@class, age, name, parent_permission)
+    @people << student
+
+    puts 'Student created successfully'
```
Comment (jaflih) [inline main.rb]: Please correct the bug, I can' create a student
Capture d’écran 2022-11-05 à 13 48 59

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Specialization: '
+    specialization = gets.chomp
+
+    teacher = Teacher.new(specialization, age, name)
+    @people << teacher
+
```
Comment (jaflih) [inline main.rb]: Please correct the bug, I can' create a teacher
Capture d’écran 2022-11-05 à 13 49 31

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @killy10o10 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
Nice commits messages✔️
Descriptive PR ✔️
Good use of GitHub flow ✔️
Clean Code✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @killy10o10, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**